PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
@bjd2385 No worries. You clearly know more about type systems than I do. 

There's a lot of history in this code base, and some parts of it look strange until you try to work out what it would take to modernize it without breaking everyone else's code. Sometimes updating works, sometimes it doesn't. (tiff metadata is a poster child for that)